src/views/AddBirthdayForm.vue

```diff
@@ -91,9 +91,14 @@ const errors = reactive({
   website: '',
 })
 
+// TODO: IDK how to share this function with the backend
+function isValidISODate(date: string): boolean {
+  return /^\d{4}-\d{2}-\d{2}$/.test(date) == true && isNaN(Date.parse(date)) == false
+}
```

Comment:
Using the same implementation as the backend function would be nice. But my JavaScript skills are too rusty to figure out how to modularise things here.